ansible/deploy.yml

```diff
@@ -98,7 +98,6 @@
           paths: "{{ packages_path }}"
           patterns:
             - .*aeternity-.*ubuntu.*.tar.gz
-            - .*epoch-.*ubuntu.*.tar.gz
```

Comment:
Can this safely be removed? When this was added, the idea was that there  still might be some old downloads on machines. Files downloaded before epoch was renamed.

Reply:
We don't deploy this name some time already, I doubt there is a leftovers, even it is it's probably a single file.